CHANGELOG.md

```diff
@@ -45,6 +46,7 @@ The format is based on the [KeepAChangeLog] project.
 [#532]: https://github.com/OpenIDC/pyoidc/pull/532
 [#498]: https://github.com/OpenIDC/pyoidc/issues/498
 [#534]: https://github.com/OpenIDC/pyoidc/pull/534
+[#549]: https://github.com/OpenIDC/pyoidc/pull/XXX
```

Comment:
should have pull/549 here instead of pull/XXX

Reply:
This should also be moved to a different section, as 0.14.0 is already released.